### ISSUE TITLE
after which step git push is executed?

### ISSUE THREAD

**Issue by DrVanScott** [researchgate/gradle-release#152]
Hi

In the introduction part of the README.md the steps of the plugin are summarized, but the information about the exact moment of "git push" is missing. From my perspective this should be the very last action in the sequence - only if it runs without error so far? Do you agree?

Anyway, could this be documented?

**Comment by danez** [comment]
Unfortunately this is not the case, as we support multiple version control systems. So when the tag or the commits are created they are immediately pushed.

**Comment by swpalmer** [comment]
I agree that this should be the case.  As far as I know in any system that requires a 'push' in addition to a 'commit', the push should always be the last thing.  Is this not compatible with the supported VCSs?

**Comment by danez** [comment]
I agree that it is not optimal right now, and that it can be improved. But it is not on my top prio list.

**Comment by glerup** [comment]
Would you accept a PR with this piece of functionality?
